README.md

```diff
@@ -1,2 +1,91 @@
-# tw-academy-s9
-This is my repository for the Lesson 9 - TW Academy
+# Sample Style Guide
+
+## Technical Writing Academy - May 2021
+
+## Table of Contents
+
+* __Bold__
```

Comment:
This section is not an unordered list. It rather is a Table of Contents with links to the corresponding sections.

You can link sections in this way:

* [Bold](#Bold)
* [Italics](#Italics)
* [Fixed Font](#Fixed-Font)
* ....